tests/page-objects/forms/generic-form.wdio.page.js

```diff
@@ -8,7 +11,31 @@ const nextPage = async (numberOfPages = 1) => {
   }
 };
 
+const formByHref = (href) => {
```

Comment:
I think this is a little misleading. We're actually passing a form id, not a "href" here. 
I think we should rename this to: `getFormLinkElementById` to be extra clear on what it does.